plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/core.py

```diff
@@ -13,6 +13,8 @@
 from hydra.plugins.sweeper import Sweeper
 from hydra.types import TaskFunction
 
+# pylint: disable=logging-fstring-interpolation,no-self-used
+
 log = logging.getLogger(__name__)
```

Comment:
feel free to move the logger into the NevergradSweeper if it makes sense. no strong opinion here.